pkg/apis/componentconfig/cloudcore/v1alpha1/validation/validation.go

```diff
@@ -34,11 +34,12 @@ import (
 // ValidateCloudCoreConfiguration validates `c` and returns an errorList if it is invalid
 func ValidateCloudCoreConfiguration(c *v1alpha1.CloudCoreConfig) field.ErrorList {
 	allErrs := field.ErrorList{}
-	allErrs = append(allErrs, ValidateKubeAPIConfig(*c.KubeAPIConfig)...)
+	//allErrs = append(allErrs, ValidateKubeAPIConfig(*c.KubeAPIConfig)...)
```

Comment:
Why comments this line?

Reply:
@fisherxu will remove comment symbol. 
And func ValidateKubeAPIConfig is invalid when os is windows, need to be compatibale with windows?

Reply:
Maybe caused by this func `!path.IsAbs(k.KubeConfig)`? We can use `filepath.IsAbs`.